src/wizard/dotnet/xamarin.md

```diff
@@ -85,6 +85,30 @@ You can Verify Sentry by raising an unhandled exception. For example, you can us
 
 You might need to open the app again for the crash report to be sent to the server.
 
+### Performance Monitoring
```

Comment:
This looks odd in the file - can you confirm it's displaying as an H3? I don't know what the issue is (apologies), but it doesn't seem to be reading as a heading level.

Reply:
They show like this today:

![image](https://user-images.githubusercontent.com/1633368/121072987-f93a3100-c79f-11eb-9d4a-f30e691486a3.png)